### PR TITLE
[LM-12677] Local DB Notification Stack

### PR DESCRIPTION
**Pull request by usmansalimLM**
## Description

**Features**
- Notification stack through localDB.
- Added notifications for DM requests.

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: example/App.tsx [hunk 1]
```diff
@@ -5,6 +5,11 @@
  * @format
  */
 
+
+if (__DEV__) {
+  require("./ReactotronConfig");
+}
+
```
Comment (jaixchaudhary) [inline example/App.tsx:12]: remove this

File: example/pushNotification/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Alert, Platform } from "react-native";
+import { getUniqueId } from "react-native-device-info";
+import { validateRegisterDeviceRequest } from "./registerDeviceApi";
+import messaging from "@react-native-firebase/messaging";
+
+export const pushAPI = async (fcmToken: string, accessToken: string) => {
+  const deviceID = await getUniqueId();
+
+  console.log("received",fcmToken,"aaaaanndddd",accessToken,"DEVICE ID",deviceID)
+
```
Comment (jaixchaudhary) [inline example/pushNotification/index.tsx:10]: remove logs

File: example/index.js [hunk 3]
```diff
@@ -6,6 +6,71 @@ import App from './App';
...
+              ))
+            },5000)
+          }
+        } else {
+          setTimeout(() => {
+            navigationRef.dispatch(StackActions.push(
+              routes?.route,
+              routes?.params
+            ))
+          },5000)
```
Comment (jaixchaudhary) [inline example/index.js:68]: can we try to make it less like 2s or something?

File: likeminds-chat-reactnative-integration/ChatSX/context/ChatroomContext.tsx [hunk 4]
```diff
@@ -695,7 +695,7 @@ export const ChatroomContextProvider = ({ children }: ChatroomContextProps) => {
       if (navigationFromNotification) {
         if (appState.match(/active|foreground/)) {
           // App has gone to the background
-          await fetchInitAPI();
+          // await fetchInitAPI();
```
Comment (jaixchaudhary) [inline likeminds-chat-reactnative-integration/ChatSX/context/ChatroomContext.tsx:698]: remove unnecessary code, if it is not used in future

Comment (usmansalimLM) [reply likeminds-chat-reactnative-integration/ChatSX/context/ChatroomContext.tsx:698]: left it for future reference in case of any bug or weird behaviour

**Review by jaixchaudhary — APPROVED** [review]

**Pull request merged**